pkg/webui/lib/errors/utils.js

```diff
@@ -328,3 +346,27 @@ export const toMessageProps = error => {
 
   return props
 }
+
+/**
+ * `ingestError` provides a unified error ingestion handler, which manages
+ * forwarding to sentry and other logic that should be applied when errors
+ * occur. The error object is not modified.
+ *
+ * @param {object} error - The error object.
+ * @param {object} extras - Sentry tags to be sent.
+ * @param {object} tags - Sentry extras to be sent.
+ */
+export const ingestError = (error, extras = {}, tags = {}) => {
```

Comment:
```suggestion
export const ingestSentryError = (error, extras = {}, tags = {}) => {
```

as it mentions sentry in jsdoc above

Reply:
The jsdoc says 

> which manages forwarding to Sentry **and other logic** **that should be applied** when errors occur.

So it's not exclusively about Sentry, right now it also handles logging the error (in dev mode).